react/react2/week1/github-user-seracher/src/ReposContext.jsx

```diff
@@ -0,0 +1,22 @@
+import  {useEffect, useState, createContext} from 'react';
+
+ export const ReposContext = createContext()
+
+const ReposProvider = ({children}) => {
+    const [repos, setRepos] = useState([])
```

Comment:
You could move this state to the Context.js file and consume it here.

Reply:
Yes thats good idea 👍 💡  Thanks for highlighting this 🙈